src/commands/neuron_manage.rs

```diff
@@ -407,15 +408,27 @@ pub fn exec(auth: &AuthInfo, opts: ManageOpts) -> AnyhowResult<Vec<IngressWithRe
 
     if let Some(proposals) = opts.register_vote {
         for proposal in proposals {
-            let args = Encode!(&ManageNeuron {
-                id,
-                command: Some(Command::RegisterVote(RegisterVote {
-                    vote: if opts.reject { 2 } else { 1 },
-                    proposal: Some(ProposalId { id: proposal }),
-                })),
-                neuron_id_or_subaccount: None,
-            })?;
-            msgs.push(args);
+            let mut proposals = Vec::new();
+            if proposal.contains('-') {
+                let (mut first, last) = get_range(&proposal)?;
+                proposals.resize_with((last - first + 1) as usize, || {
+                    first += 1;
+                    first - 1
+                });
```

Comment:
```suggestion
                proposals = (first, last+1).collect();
```
should do the same, but is more obviously correct (at least to me)